e2e/cypress/integration/ui/settings/data-feed/data-feed-overview.spec.ts

```diff
@@ -0,0 +1,120 @@
+describe('chef datafeed', () => {
+
+    const token = 'behwveh3238238=';
+    const DestinationID = 'chef-server-dev-test';
+    const reusableDate = Date.now();
+    let adminIdToken = '';
+    const name = 'cytest' + reusableDate;
+    const url = 'https://localhost/api/x_chef_automate/asset';
+    const ServiceNow = 'Service Now';
+    const Webhook = 'Webhook';
+    before(() => {
+        cy.adminLogin('/').then(() => {
+            const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+            adminIdToken = admin.id_token;
+            cy.get('app-welcome-modal').invoke('hide');
+            cy.restoreStorage();
+            cy.get('body').type('feat');
+            cy.get('.title').contains('Chef Automate Data Feed').parent().parent()
+              .find('.onoffswitch').click();
+            cy.get('chef-button').contains('Close').click();
+            cy.reload();
+            cy.contains('know').click();
+        });
+        cy.restoreStorage();
+      });
+
+    beforeEach(() => {
+        cy.restoreStorage();
+      });
+
+    afterEach(() => {
+        cy.saveStorage();
+    });
+    describe ('chef data feed page for Service Now', () => {
+        before(() => {
+            cy.request({
+                auth: { bearer: adminIdToken },
+                method: 'POST',
+                url: '/api/v0/secrets',
+                body: {
+                  name: 'anyname',
+                  type: 'data_feed',
+                  data: [
+                      {
+                          key: 'headers',
+                          value: '{\"Authorization\":\"Basic service-now-token\"}'
+                      },
+                      {
+                          key: 'auth_type',
+                          value: ''
+                      }
+                  ]
+                }
+            }).then((secrectResp) => {
+                  expect(secrectResp.status).to.equal(200);
+                  expect(secrectResp.body).to.property('id');
+
+                  cy.request({
+                      auth: { bearer: adminIdToken },
+                      method: 'POST',
+                      url: '/api/v0/datafeed/destination',
+                      body: {
+                          url: url,
+                          name: name,
+                          secret: secrectResp.body.id,
+                          services: ServiceNow,
+                          integration_types: Webhook
+                      }
+                  }).then((dataFeedResp) => {
+                    expect(dataFeedResp.status).to.equal(200);
+                    expect(dataFeedResp.body).to.property('id');
+
+                    cy.visit(`/settings/data-feeds/${dataFeedResp.body.id}`);
```

Comment:
```suggestion
                    cy.visit(`/settings/data-feeds/${dataFeedResp.body.id}`);
                    cy.get('app-welcome-modal').invoke('hide');
```
Cypress is failing because we dont have this line